<robot name="NooBot">
  <link name="base_footprint"></link>

  <joint name="base_joint" type="fixed">
    <parent link="base_footprint" />
    <child link="base_link" />
    <origin xyz="0 0 0.13" rpy="0 0 0" />
  </joint>
  
  <link name="base_link">
    <visual>
      <origin xyz="0 0 0" rpy="0 0 0" />
      <geometry>
        <cylinder radius="0.2" length="0.12" />
      </geometry>
      <material name="base_link-material">
        <color rgba="0.6375968739867731 0.2541520943200296 0.5972017883558645 1" />
      </material>
    </visual>

  </link>


  <joint name="base_link_to_left_link" type="fixed">
    <parent link="base_link" />
    <child link="left_link" />
    <origin xyz="0 0.2 -0.07" rpy="1.5707963267948963 0 0" />
  </joint>


  <link name="left_link">
    <visual>
      <origin xyz="0 0 0" rpy="0 0 0" />
      <geometry>
        <cylinder radius="0.063" length="0.04" />
      </geometry>
      <material name="left_link-material">
        <color rgba="0.3277780980458375 0.34670405634441115 0.22696587349938613 1" />
      </material>
    </visual>

  </link>


  <joint name="base_link_to_right_link" type="fixed">
    <parent link="base_link" />
    <child link="right_link" />
    <origin xyz="0 -0.2 -0.07" rpy="1.5707963267948963 0 0" />
  </joint>


  <link name="right_link">
    <visual>
      <origin xyz="0 0 0" rpy="0 0 0" />
      <geometry>
        <cylinder radius="0.063" length="0.04" />
      </geometry>
      <material name="right_link-material">
        <color rgba="0.3277780980458375 0.34670405634441115 0.22696587349938613 1" />
      </material>
    </visual>

  </link>
</robot>